authentication-service/auth-service-deployment.yml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: auth-service-v1
+  namespace: default
+  labels:
+    app: auth-service
+    version: v1
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: auth-service
+      version: v1
+  template:
+    metadata:
+      labels:
+        app: auth-service
+        version: v1
+    spec:
+      containers:
+        - name: auth-service
+          image: cichan/authentication-service:prod
```

Comment:
should not `prod`, `test` or something. 

it should fixed version in [semver.org](https://semver.org/) format

Reply:
fixed